testing/gogs/APKBUILD

```diff
@@ -1,5 +1,5 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
-# Maintainer:
+# Maintainer: 7heo <7heo@mail.com>
 pkgname=gogs
 pkgver=0.6.1
 pkgrel=4
```

Comment:
You must always bump `pkgver`, or `pkgrel` when doing any change in the abuild. That’s why CI fails when no package is build, to prevent this kind of mistakes.